channels/audin/client/opensles/audin_opensl_es.c

```diff
@@ -161,7 +161,9 @@ static void audin_opensles_free(IAudinDevice* device)
 
 static BOOL audin_opensles_format_supported(IAudinDevice* device, audinFormat* format)
 {
+#ifdef WITH_DEBUG_DVC
 	AudinOpenSLESDevice* opensles = (AudinOpenSLESDevice*) device;
+#endif
 	
 	DEBUG_DVC("device=%p, format=%p", opensles, format);
```

Comment:
@realjiangms nope there shouldn't be an error. Have a look to
channels/audin/client/audin_main.h around line 35.